js/bootstrap-datepicker.js

```diff
@@ -1062,7 +1062,8 @@
 				}
 
 				clsName = $.unique(clsName);
-				html.push('<td class="'+clsName.join(' ')+'"' + (tooltip ? ' title="'+tooltip+'"' : '') + '>'+prevMonth.getUTCDate() + '</td>');
+				console.log(this.o.dateCells);
```

Comment:
Looks like that it should be removed. Build is broken in Travis because console object not found.